v2/types/index.ts

```diff
@@ -1,5 +1,13 @@
+import { RawChain } from "picklefinance-core/lib/chain/Chains";
 import { FC } from "react";
 
 export interface PickleFinancePage extends FC {
   PageTitle: FC;
 }
+
+export interface JarDetails {
+  name: string;
+  apiKey?: string;
+  chain: string;
+  apy: number;
+}
```

Comment:
And one more heads up - I started this file to include files until it becomes big and we find ways to organize them better. Good to append for now but let's keep our eyes open for better organization of code as this list grows.